### DISCUSSION TITLE
Full Combo animation before the result panel

### DISCUSSION THREAD

**Discussion by Nope10** [ppy/osu#13138] [Ideas]
**Describe the new feature:**
Add a "Full Combo" text animation with sound (5 or more seconds) when someone FC a map before showing the result screen (like on other popular rythm games and just for the style). 

If people don't like it, they can choose to turn it off in the options.

**Proposal designs of the feature:** 
A (customisable please) .png image and .wav .mp3 sound (customisable too)

**Comment by recognitionmark** [comment]
So this means that I can make a Fullcombo image and put someone shouting "Full Combo!" in the background? I think it's quite time-consuming.
